sos/plugins/kdump.py

```diff
@@ -27,13 +27,71 @@ class RedHatKDump(KDump, RedHatPlugin):
     files = ('/etc/kdump.conf',)
     packages = ('kexec-tools',)
 
+    def kdump_parse_fs(self):
+        device = None
+        for line in self.kdumpconf:
+            if line.startswith(("ext2", "ext3", "ext4", "nfs", "xfs")):
+                try:
+                    device = line.split()[1]
+                except (IndexError):
+                    return False
+        if device:
+            try:
+                fp = open('/etc/fstab', 'r')
+                lines = fp.readlines()
+                fp.close()
+            except IOError:
+                return False
+
+            for line in lines:
+                if line.startswith((device)):
+                    try:
+                        self.fs = line.split()[1].rstrip('/')
+                        return True
+                    except (IndexError):
+                        return False
+        return False
+
+    def kdump_parse_path(self):
+
+        for line in self.kdumpconf:
+            if line.startswith("path"):
+                try:
+                    self.path = line.split()[1]
+                    return True
+                except (IndexError):
+                    return False
+
+        return False
+
+    def read_kdump_conffile(self):
+        try:
+            fp = open('/etc/kdump.conf', 'r')
+            self.kdumpconf = fp.readlines()
+            fp.close()
+            return True
+
+        except IOError:
+            return False
```

Comment:
Ditto for boolean returns. You can also simplify the IO by using a `with` block.